src/dashboard/widgets/widgets.js

```diff
@@ -67,6 +67,7 @@ const Widgets = () => {
           <button style={getTabButtonStyle('tab1')} onClick={() => handleTabChange('tab1')}>User Management</button>
           <button style={getTabButtonStyle('tab2')} onClick={() => handleTabChange('tab2')}>Role Management</button>
           <button style={getTabButtonStyle('tab3')} onClick={() => handleTabChange('tab3')}>Access Key Management</button>
+          <button style={getTabButtonStyle('tab4')} onClick={() => handleTabChange('tab4')}>User Access Key Management</button>
```

Comment:
This should be in the user profile section. Since it is for a specific user and not for the app administrators.
But if this is problematic we can leave it here.

Reply:
i will leave it here for now, but will open an issue to improve profile page overall